fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1535,6 +1564,18 @@ public long loadHeader(DataInputStream dis, long checksum) throws IOException {
         return newChecksum;
     }
 
+    public long loadHeaderV2(DataInputStream dis, long checksum) throws IOException {
```

Comment:
need to add all the loadxxxV2 to loadImage method?

Reply:
will be added in another PR to avoid conflict